src/app/utils/field-validation/answerField.factory.ts

```diff
@@ -0,0 +1,24 @@
+import { BasicField, IColumnSchema } from '../../../types'
+
+type ColumnWithFieldProperties = IColumnSchema & {
+  getQuestion: { (): string }
+  description: string
+  disabled: boolean
+  fieldType: BasicField.ShortText | BasicField.Dropdown
+}
+
+/**
+ * Takes a table field column and generates a form field by
+ * filling in the missing attributes.
+ * */
+export const createAnswerFieldFromColumn = (
+  column: IColumnSchema,
+): ColumnWithFieldProperties => {
+  const columnField = column
+  ;(columnField as ColumnWithFieldProperties).disabled = false
+  ;(columnField as ColumnWithFieldProperties).description = 'some description'
+  ;(columnField as ColumnWithFieldProperties).fieldType = column.columnType
+  ;(columnField as ColumnWithFieldProperties).getQuestion = () =>
```

Comment:
Avoid this sort of data mutation, which is a code smell:

https://blog.sapegin.me/all/avoid-mutation/